src/pages/api/jobPosting/getAll.ts

```diff
@@ -0,0 +1,26 @@
+// Third-party import
+import { NextApiRequest, NextApiResponse } from "next";
+// Local import
+import { getAllPosting } from "@/backend/actions/jobPosting";
+
+export default async function hanlder(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
Same thing here too but not sure if it's on purpose

Reply:
do it once, its a mistake, do it twice, its intended

Reply:
sorry my bad and nice catch

Reply:
ur bad at spelling